plugins/generator-1.20.4/neoforge-1.20.4/templates/elementinits/enchantments.java.ftl

```diff
@@ -38,10 +38,10 @@ package ${package}.init;
 
 public class ${JavaModName}Enchantments {
 
-	public static final DeferredRegister<Enchantment> REGISTRY = DeferredRegister.create(ForgeRegistries.ENCHANTMENTS, ${JavaModName}.MODID);
+	public static final DeferredRegister<Enchantment> REGISTRY = DeferredRegister.create(BuiltInRegistries.ENCHANTMENT, ${JavaModName}.MODID);

```

Comment:
I'd use `Registries.ENCHANTMENT` instead, for consistency

Reply:
What is the difference between these two classes?

Reply:
Hm, they provide examples of using both to create registries, weird

Reply:
Actually no, they use Registries for creating a registry, and Builtinregistries for getting values from the registries.